app/league/[leagueId]/entries/[entryId]/week/Week.tsx

```diff
@@ -110,44 +123,61 @@ const Week = ({ league, NFLTeams, week }: IWeekProps): JSX.Element => {
   };
 
   useEffect(() => {
+    if (!selectedLeague) {
+      getSelectedLeague();
+      return;
+    }
     getSchedule(week);
-  }, [week]);
+    setIsLoading(false);
+  }, [week, selectedLeague]);
 
-  if (schedule.length === 0) {
+  if (schedule.length === 0 || isLoading) {
     return <p>Loading...</p>;
   }
 
   return (
-    <section className="w-full pt-8" data-testid="weekly-picks">
-      <h1 className="pb-8 text-center text-[2rem] font-bold text-white">
-        Your pick sheet
-      </h1>
-
-      <FormProvider {...form}>
-        <form
-          className="mx-auto flex w-[90%] max-w-3xl flex-col items-center gap-8"
-          onSubmit={form.handleSubmit(onSubmit)}
+    <div>
+      <nav className="py-6 text-orange-500  hover:no-underline">
+        <LinkCustom
+          className="text-orange-500 flex gap-3 items-center font-semibold text-xl hover:no-underline"
+          href="/league/all"
```

Comment:
Does the `hover:bg-zinc-800 transition` from the WeeklyPickButton apply to this still when hovering over it on WeekTeams?

Reply:
IIRC it did in my testing, but now you're making me question myself! I'll confirm tomorrow (7/9/24) and report back.